jinja2/includes/common_macros.html

```diff
@@ -37,7 +37,10 @@
       <nav>
         <a href="{{ user.get_absolute_url() }}">{{ _('Profile') }}</a>
         |
-        <a href="{{ url('account_logout') }}">{{ _('Sign Out') }}</a>
+        <a id="logout-button" data-logout-url="{{ url('allauth_account_logout') }}"
```

Comment:
This may need to be a form to implement the POST correctly.  However a form may break layout, so maybe there needs to be a hidden form that gets called from JS.